doc/source/api/ast_class.rst

```diff
@@ -1,7 +1,7 @@
 AST tree
 ========
 
-.. currentmodule:: pydita_ast.ast_tree
+.. currentmodule:: pyconverter.xml2py.ast_tree
 
 The following classes are used to create the AST tree:
```

Comment:
Can we define AST tree here--because AST wasn't a term any of us writers were familiar with. Does it stand for abstract syntax tree? If so, maybe begin this section with this:

An AST (abstract syntax tree) is an abstract syntactic structure of text. Each node of the tree denotes a construct occurring in the text. Every element the AST contains can be edited and enhanced with information such as properties and annotations.

Reply:
Thank you @PipKat. 
I made a change on this to be more accurate. Let me know if you want me to make a modification.